main.go

```diff
@@ -82,3 +78,15 @@ func (depper *Depper) registerIngestorStream(ingestor ingestors.StreamingIngesto
 		}
 	}()
 }
+
+func setupLogger() {
+	log.SetFormatter(&log.TextFormatter{
```

Comment:
This seemed to control the logger globally yeah?

Reply:
Yeah, I'm still using the singleton logger here -- cool thing is that we can switch to the [Zalgo logger](https://github.com/sirupsen/logrus#formatters) if we want 😃